src/appleseed.python/bindblendermeshconverter.cpp

```diff
@@ -130,45 +124,90 @@ void convert_bl_mesh(
                 face.v[1],
                 face.v[2],
                 face.mat_nr));
+        if (face.v[3] != 0)
+        {
+            blender_mesh->push_triangle(
+                Triangle(
+                    face.v[2],
+                    face.v[3],
+                    face.v[0],
+                    face.mat_nr));
+        }
     }
 
+    size_t bl_face_index = 0;
```

Comment:
Can't it be initialized directly in the for loop ? Like this

Reply:
`         for (size_t tri_index = 0, bl_face_index = 0; tri_index < triangle_length; ++tri_index, ++bl_face_index) `